test/testdata/lsp/rename/locals.rb

```diff
@@ -0,0 +1,103 @@
+# typed: true
```

Comment:
This is just a note for future reference, not a request to action in this PR, but

For `.rbedited` tests, I usually prefer making many small tests rather than one large test, because in the case where someone wants to e.g. make a small change to a single portion of this test, it now requires updating 10 extra files.